projects/canopy/src/lib/hide-at/hide-at.directive.ts

```diff
@@ -0,0 +1,30 @@
+import { Directive, ElementRef, Input, Renderer2 } from '@angular/core';
+
+import { Breakpoints } from '../shared/breakpoints.interface';
+
+@Directive({
+  selector: '[lgHideAt]',
+})
+export class LgHideAtDirective {
+  responsiveUtilClass: string;
+  @Input()
+  set lgHideAt(breakpoint: Breakpoints) {
```

Comment:
You could trial [this approach](https://github.com/Legal-and-General/canopy/discussions/303#discussioncomment-598758). I realise that it would be a shift in approach and I wouldn't hold PR up for it.

Reply:
Thanks, yes I'd prefer to get this one in if that's okay. That discussion was born from working on this branch where I've started to wonder if we could improve it 👍